EOG In Practice: CW21 (#79)
softwaresaved/event-organisation-guide: docs/eog/eog-in-practice/cw21/event-project-stage/eps-risk-management.rst

2. It was an opportunity to improve and build off of what we achieved running :ref:`CW20 Online <In-Practice-CW20>`.

It turned out that there were still many restrictions around international travel and in-person events at the time of CW21, so we successfully mitigated the risk of having to cancel another in-person event. 

The main stakeholder
--------------------

The risks that we identified and needed to mitigate relating to the main stakeholder - in this case the Software Sustainability Institute - were:

- Event goals and objectives not aligned with the main stakeholder goals
- Damage to the main stakeholder’s reputation

As we demonstrated through CW20, the event goals and objectives can be achieved through a virtual event.
To ensure that the :ref:`cw21-fs-goals-and-objectives` aligned with the main stakeholder goals, the Director was consulted during the :ref:`cw21-feasibility-stage`, and both the Director and Research Director were invited to participate on the :ref:`CW21 Steering Committee <cw21-eps-committees>`.

> Running CW20 virtually at the start of the pandemic led to the Institute being internationally recognised and regularly consulted on virtual event organisation and community building. 

Author: There are links on lines 15 and 19? 

The risks for CW21 associated with the main stakeholder are that the goals and objectives are not aligned with the main stakeholder or damage to reputation (from the main chapters of the EOG). These paragraphs are showing how these risks have been mitigated through the experience of CW20 - that we can still achieve our goals through online events (as demonstrated by CW20) and that the institute's reputation will not be damaged (as the opposite was demonstrated by CW20). 

I can add the following clarification: 

```suggestion
As we demonstrated through CW20, the main stakeholder’s reputation would not be damaged by running a virtual event.
In fact, by running CW20 virtually at the start of the pandemic, the Institute was internationally recognised and regularly consulted on virtual event organisation and community building. 
```

> 70% of CW20 participants who gave feedback stated that "CW20 is the best [online] event I have attended".

Author: ```suggestion
Additionally, 70% of CW20 participants who gave feedback stated that "CW20 is the best [online] event I have attended".
```

.. figure:: ../img/CW20_participant_feedback1.png
  :alt: CW20 Participant Feedback on how the event compared with other online events 
  
  Feedback from CW20 participants on how the event compared with other online events

> Due to the attention and care that we put into improving and building off of what we achieved during CW20, we had similar feedback after CW21 when compared with the past year of virtual events. 

Author: ```suggestion
Therefore, due to the attention and care that we put into improving and building off of what we achieved during CW20 and mitigating the risk of damaging the Institute's reputation, we had similar feedback after CW21 when compared with the past year of virtual events. 
```

- Limit the number of registrations to 150 so that we could manage the interactive sessions and have a more intimate event

Within the CW20 feedback form, we asked participants how they would prefer us to organise CW in the future (in-person or online).
Half of respondents said that they didn't mind how it was run, and 33% said that they prefer an in-person event.

.. figure:: ../img/CW20_participant_feedback2.png
  :alt: CW20 Participant Feedback on how to organise the event in the future
  
  Feedback from CW20 participants on how to organise the event in the future 

However, as the effects of the pandemic were ongoing, we decided that CW21 would take place online.
As the vaccination programmes advance and things begin to open up, there is pressure and expectation for hybrid events. 
We asked CW21 on the feedback form to rank how they prefer future CWs to be organised: in-person, online, hybrid or don't mind, the results of which are displayed below:

.. figure:: ../img/CW21_participant_feedback2.png
  :alt: CW21 Participant Feedback on how to organise the event in the future 
  
  Feedback from CW21 participants on how to organise the event in the future

> 

Author: ```suggestion

Opinions were more or less evenly split between the different options, and we believe this may be due to ongoing uncertainty about the pandemic.
```

Keynote speakers
^^^^^^^^^^^^^^^^

> Risks that we identified from running CW20 online included sickness or being unable to present due to caring or other responsibilities imposed based on restrictions due to the pandemic.

Author: No, this should be CW20 as this was a lesson learned from CW20 to mitigate against in CW21.

Connection or technical issues could also arise.
The strategy we took to mitigate these risks was to request the keynote speakers to pre-record their presentations. 
We provided a speaker guide with guidance and advice for recording their talk, and our full support with editing and captioning the videos. 

This strategy had the added benefit that it allowed us to have keynote speakers from regions with time zones incompatible with the times of the event - the speakers did not have to present outside of their normal working hours. 

Session chairs
^^^^^^^^^^^^^^^

The risks identified regarding the session chairs included connection or other technical issues. 

> We also identified sickness or being unable to present due to caring or other responsibilities imposed based on restrictions due to the pandemic as risks from running CW20 online.

Author: This was a lesson learnt from CW20 that we applied to CW21 risks. I will reword that we identified this risk from running the event online the previous year.
